### DISCUSSION TITLE
The old searcher

### DISCUSSION THREAD

**Discussion by kazcw** [enso-org/enso#5909] [Help and troubleshooting] [answered]
What is the status of the old searcher? I was trying to port it to the new doc-parser interface, and it doesn't work to begin with: `enso -feature-preview.new-component-browser=false` doesn't show any docs. Is looks like it's just dead code waiting to be removed (#5661)?

**Comment by vitvakatu** [comment, marked as answer]
Yes, we want to remove it completely, you can break it without consequences. The only thing that stops us right now is that removing it properly requires quite a lot of changes in the existing code, and we don't see a good reason to spend time on that right now.
